tests/sentry/grouping/test_benchmark.py

```diff
@@ -27,10 +30,10 @@ def benchmark_available() -> bool:
     sorted(set(CONFIGURATIONS.keys()) - {NO_MSG_PARAM_CONFIG}),
     ids=lambda config_name: config_name.replace("-", "_"),
 )
-def test_benchmark_grouping(config_name, benchmark):
+def test_benchmark_grouping(config_name: str, benchmark: ModuleType) -> None:
```

Comment:
this is actually `benchmark: BenchmarkFixture` (from pytest-benchmark) -- though we don't actually support using pytest-benchmark so this is probably just broken / dead code

I am quite surprised it allowed `benchmark.pedantic` below without an attribute error?